### ISSUE TITLE
the feature use BatchNorm1d but amsoftmax weight use l2_norm will this difference match the paper request? 

### ISSUE THREAD

**Issue by mlinxiang** [ZhaoJ9014/face.evoLVe.PyTorch#49]
why feature not use l2_norm as weight?
![image](https://user-images.githubusercontent.com/6283983/57193479-0277c480-6f6e-11e9-8080-c80e844e9b3b.png)

**Comment by ZhaoJ9014** [comment]
> why feature not use l2_norm as weight?
> ![image](https://user-images.githubusercontent.com/6283983/57193479-0277c480-6f6e-11e9-8080-c80e844e9b3b.png)

Hi, thank you for your question. The implementation is actually same with the paper.
